src/Loops.java

```diff
@@ -0,0 +1,5 @@
+public class Loops {
```

Comment:
Probeer deze opdrachten op een later moement nog eens te doen. Meer oefening is altijd goed.